### ISSUE TITLE
solo3 without x11 support

### ISSUE THREAD

**Issue by aereinha** [ncareol/homebrew-lrose-solo3#2]
I have in a few hours removed the parts of solo3 that require x11 support. The parts that do are only used to create images to be saved, which, I and others I have talked to, never was working as intended anyways. If you remove all mention to sii_X_stuff.cc and its functions and anything that uses sii_png_image and the save image folder location menubar. It will compile and run. I can send a complete diff if you want.

Solo3 does launch however there are some draw issues occurring now where only the first sweep field frame is drawn and the images do not redraw automatically. You have to click the menubar to get a redraw of the fields. If you guys have some hints as to which gtk/gdk things might be either holding up a redraw or a change to force a redraw after a mouse click on the screen that would be appreciated. 

In summary solo3 support without x11 seems possible with relative minor adjustments needed to the redrawing of the images left to fix.

**Comment by vanandel** [comment]
Yes, please send your diffs to vanandel@ucar.edu.

**Comment by burghart** [comment]
Image saving has been refactored to use native gtkmm tools, and no longer requires X11 support. Under OS X, there appears to be a bug in the underlying support libraries which causes the red and blue channels of the saved images to be swapped.

Regardless, the 20160613 version of solo3 now builds under OS X without X11.
